docs/mathematics/sum-of-divisors.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Sum of Divisors
+slug: /mathematics/sum-of-divisors
+---
+
+## Introduction
+
+Sum of divisors refers to the sum of numbers in the range `1 to n` which perfectly divides `n`.
```

Comment:
Have a more detailed introduction. Usually follow this pattern while introducing what you are writing about: 
- Introduce the topic
- Why its relevant
- Prompt the reader to go next